### ISSUE TITLE
Bert gives low accuracy on multi-class classification task

### ISSUE THREAD

**Issue by xwang222** [google-research/bert#1124]
Hi everyone,

I am working on a small data set which:
- contains 1500 pieces of news articles. 
- All of these articles were ranked by human beings with regard to their sentiment/degree of positive on a 5-point scale. 
- clean in terms of spelling error
- average length is greater than 512 words
- balanced data set 

I regard this as a multi-class classification problem and I want to fine-tune BERT with this data set. In order to do that, I used Ktrain package and basically follows the tutorial. Below is my code:

(x_train,  y_train), (x_test, y_test), preproc = text.texts_from_array(x_train=x_train, y_train=y_train,
                                                                       x_test=x_test, y_test=y_test,
                                                                       class_names=categories,
                                                                       preprocess_mode='bert',
                                                                       maxlen= 510, 
                                                                       max_features=35000)

model = text.text_classifier('bert', train_data=(x_train, y_train), preproc=preproc)
learner = ktrain.get_learner(model, train_data=(x_train, y_train), batch_size=6)
learner.fit_onecycle(2e-5, 4)

However, I only get a validation accuracy at around 25%, which is way too low.
              precision    recall  f1-score   support

           1       0.33      0.40      0.36        75
           2       0.27      0.36      0.31        84
           3       0.23      0.24      0.23        58
           4       0.18      0.09      0.12        54
           5       0.33      0.04      0.07        24

    accuracy                            0.27       295
    macro avg            0.27      0.23      0.22       295
    weighted avg       0.26      0.27      0.25       295

    array([[30, 25, 10,  9,  1],
              [20, 30, 26,  8,  0],
              [18, 21, 14,  4,  1],
              [16, 26,  7,  5,  0],
              [ 8,  8,  5,  2,  1]])

I also tried the head+tail truncation strategy since some of the articles are pretty long, however, the performance remains the same.
 
Can anyone give me some suggestions?

Thank you very much!

Best
Xu
